src/Printer/CollectorMetadataPrinter.php

```diff
@@ -13,7 +13,7 @@
 use PhpParser\Node\NullableType;
 use PhpParser\Node\Stmt\ClassMethod;
 use PhpParser\Node\UnionType;
-use PhpParser\PrettyPrinter\Standard;
+use PHPStan\Node\Printer\Printer;
```

Comment:
PHPStan has removed the `PhpParser\PrettyPrinter\Standard` alias and now we need to use `PHPStan\Node\Printer\Printer` directly